src/main/java/com/jcabi/github/mock/MkPull.java

```diff
@@ -161,18 +216,55 @@ public void patch(
     @Override
     @NotNull(message = "JSON is never NULL")
     public JsonObject json() throws IOException {
-        final JsonObject obj = new JsonNode(
-            this.storage.xml().nodes(this.xpath()).get(0)
-        ).json();
-        final JsonObjectBuilder json = Json.createObjectBuilder();
-        for (final Map.Entry<String, JsonValue> val : obj.entrySet()) {
-            json.add(val.getKey(), val.getValue());
```

Comment:
@cvrebert you are no longer adding all key/values as before. don't you think this might be an issue?